frontend/local/src/containers/patients/detail/index.js

```diff
@@ -90,26 +93,50 @@ class PatientDetail extends React.Component {
                             {bodyMeasurements && bodyMeasurements.height ? (
                                 <div className="col-sm-4">
                                     <h5>Height</h5>
-                                    {bodyMeasurements.height} cm
+                                    <ValueWithUnitString
+                                        valueUnit="cm"
+                                        stringUnit={this.props.lengthUnit}
+                                        precision={0}
+                                        formatting={SHORT}
+                                        value={bodyMeasurements.height}
+                                    />
```

Comment:
We should try to get to the same interface with displaying values as with inputs. Patient details page should not be aware of the value unit set in config.

Reply:
done